packages/api-explorer/src/Doc.jsx

```diff
@@ -25,44 +26,161 @@ const { Operation } = Oas;
 const parseResponse = require('./lib/parse-response');
 const Content = require('./block-types/Content');
 
-const pkg = require('../package.json');
+const { version: packageVersion } = require('../package.json');
```

Comment:
Hopefully with this Webpack will stop adding our entire `package.json` into the dist. 😬